### PR TITLE
Add $__from and $__to to Macros section

### PR DESCRIPTION
**Pull request by chs820**
Added examples for the use of `$__to` / `$__from` and `${__to:date}` / `${__from:date}` in the Macros section of the README

### CODE REVIEW

**Comment by zoltanbedi** [comment]
I tried to use `${__from:date}` with a query like this `SELECT ${__from:date} as timestamp` and got an error I had to put single quotes around it to work. Maybe showing an example query with quotes could help others not to miss that as I did.

**Comment by chs820** [comment]
Updated to include new example for `${__from:date}`

**Review by zoltanbedi — APPROVED** [review]
Awesome, thanks for adding them!

**Review by baldm0mma — APPROVED** [review]
Nice! I'd rope in a docs/technical writer just to go over the language before merging, just to be safe. My fave is @imatwawana. :)

File: README.md [hunk 1]
```diff
@@ -161,10 +161,12 @@ FROM `project.dataset.table`
...
+| Macro example                         | Description                                                                                                                              |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| _$\_\_timeFilter(timeColumn)_         | Will be replaced by a time range filter using the specified name.                                                                        |
+| _$\_\_timeGroup(timeColumn,interval)_ | Will be replaced by an expression usable in GROUP BY clause.                                                                             |
+| _$\_\_from_ or _$\_\_to_              | Will be replaced by a unix millisecond representation of the time filter start or end time. e.g., `timestamp_millis($__to)`              |
+| _${\_\_from:date}_ or _${\_\_to:date}_| Will be replaced by a date (ISO 8601/RFC 3339) representation of the time filter start or end time. e.g., `SELECT DATE('${__from:date}')`|
```
Comment (imatwawana) [inline README.md:169]:
```suggestion
| Macro example                         | Description                                                                                                                              |
| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
| _$\_\_timeFilter(timeColumn)_         | Will be replaced by a time range filter using the specified name.                                                                        |
| _$\_\_timeGroup(timeColumn,interval)_ | Will be replaced by an expression usable in the GROUP BY clause.                                                                             |
| _$\_\_from_ or _$\_\_to_              | Will be replaced by a Unix millisecond representation of the time filter start or end time. For example, `timestamp_millis($__to)`              |
| _${\_\_from:date}_ or _${\_\_to:date}_| Will be replaced by a date (ISO 8601/RFC 3339) representation of the time filter start or end time. For example, `SELECT DATE('${__from:date}')`|
```

**Review by imatwawana — APPROVED** [review]
A couple style nits, but otherwise looks good!

**Pull request merged**